angular/src/app/directives/leaflet/leaflet.directive.ts

```diff
@@ -0,0 +1,41 @@
+import { Directive, ElementRef, Input, NgZone } from '@angular/core';
```

Comment:
Persoonlijk zou ik hier gewoon een component van maken, het is echt een op zichzelf staand iets namelijk. Daarnaast is het nu in de directory `containers` geplaatst, maar dat is het om dezelfde reden eigenlijk ook niet. Misschien nu ook gelijk een 'eigen' directory en module geven?